spec/i18n/placeholders_spec.rb

```diff
@@ -0,0 +1,45 @@
+describe "Placeholders in strings" do
+  it "translations preserve placeholders in strings" do
+    dirs = [Rails.root.join('locale')]
```

Comment:
@durandom @bdunne This will be interesting with plugins...that is, plugins should probably end up providing translations for *their* stuff.